php74/php.ini

```diff
@@ -6,6 +6,8 @@ max_execution_time = 240
 upload_max_filesize = 200M
 post_max_size = 200M
 
+html_errors = 0
```

Comment:
this change still makes sense after xdebug3 https://github.com/skilld-labs/docker-php/pull/65